lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java

```diff
@@ -120,11 +123,17 @@ void should_CreateTaskComment_When_CopyingAnotherComment() throws Exception {
         taskService.getTaskComments(taskWithComments.getId());
     assertThat(taskCommentsAfterInsert)
         .containsExactlyInAnyOrder(comment1, comment2, taskCommentToCreate);
+    assertThat(taskService.getTask(taskWithComments.getId()).getNumberOfComments()).isEqualTo(3);
+
+    // Deleting the comment so that the comments remain the same in different tests inside this
```

Comment:
Why did you add this? How did it work before?

Reply:
sometimes it threw an error for other tests if i ran it without deleting the comment again.

Reply:
Ok

Reply:
Typically it is a good practice to use different test data in this case to prevent those dependencies. Would it be much effort to create a seperate set of test data for this test?

Reply:
No, it wouldn't be too much effort. I will do as you suggested.